fdbserver/Ratekeeper.actor.cpp

```diff
@@ -698,6 +798,48 @@ void Ratekeeper::updateRate(RatekeeperLimits* limits) {
 		break;
 	}
 
+	if (configuration.blobGranulesEnabled) {
```

Comment:
Are there potential races between enabling/disabling blob granules and checking the configuration both in monitorBlobWorkers and here?
If that's possible, it might be simpler to just set the blob worker rate to infinity when disabled or something?

Reply:
The limit starts at infinite each time updateRate is called, and then it must be lowered each time, so immediately after disabled blobGranules it will stop throttling